nuxeo-features/preview/nuxeo-preview-core/src/test/java/org/nuxeo/ecm/platform/preview/tests/adapter/TestPreviewAdapter.java

```diff
@@ -152,4 +156,18 @@ public void testFolderishDocument() throws Exception {
         assertNotNull(adapter);
     }
 
+    @Test
+    public void testConvertTextToHtml() throws IOException {
+        Blob blob = new StringBlob("thisismytest", "text/plain");
+        DocumentModel doc = session.createDocumentModel("File");
+        doc.setPropertyValue("file:content", (Serializable) blob);
+        // convert using SimpleBlobHolder (like ConvertAdapter)
+        BlobHolder blobHolder = new SimpleBlobHolder(blob);
+        BlobHolder res = conversionService.convertToMimeType("text/html", blobHolder , null);
```

Comment:
can we pass an empty map (`Map.of`, `Collections.emptyMap()`) instead of `null` on the map parameter

Reply:
`null` is accepted and turned into an empty map later on in the call stack.